wmn-data.json

```diff
@@ -4040,6 +4064,18 @@
         "cat" : "shopping",
         "valid" : true
        },
+       {
+        "name" : "ogu.gg",
+        "uri_check" : "https://ogu.gg/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "Message",

```

Comment:
Can you please replace this with something more complicated? the word "message" could give us false positives.